PlayerForceHD/SoftDocLinker
# Implement base UI

### Description
Provide a simple implementation of a basic layout on which the application can be implented on.

### Issue
#11 

### Test Scenarios
What is the expected behavior that is expected by the changes of the PR?
 - Start the development server
- Open SoftDocLinker in the browser
- A basic layout styled using the material design should open.

The layout should:

- [x] Have a large white SoftDocLinker brand in the navigation bar
- [x] A home button that is completly white on the right button area. (Icon: House, Text: "Home")
- [x] A floating action button in blue with a spinning circle (loading: true) and or a circular arrow (loading: false)
- [x] An empty navigation drawer on the left side
- [x] An empty content area on the rest of the screen
